spec/hackerone/client/report_spec.rb

```diff
@@ -213,4 +213,29 @@
       expect(scope.asset_type).to eq("url")
     end
   end
+
+  describe "#on_state_change_hooks" do
+    after do
+      described_class.clear_on_state_change_hooks
+    end
+
+    it "triggers hooks on state changes" do
+      described_class.add_on_state_change_hook ->(_report, _old_state, _new_state) do
```

Comment:
Is this just to demonstrate the multiple hooks run? If so, I think this needs a comment or else it might get deleted on future overzealous code cleanups.

Reply:
Yup, that was the intent! Although this is borderline paranoid, so I'd be okay with just removing it.

Reply:
I think it's a good thing to test